setup a11y info pages on all three themes

StephDriver opened the pull request:
closes #4694

StephDriver commented on src/core/views.py:
```diff
@@ -1091,6 +1091,25 @@ def settings_index(request):
+                # Non-press-dependent settings are always included
+                filtered_settings.append(setting)
+
+        settings = filtered_settings
```
I did wonder about this for some time while working on the branch. My feeling was that in this specific instance, no or at least not yet. because I think the accessibility bit needs to be rolled out more slowly. Likely first to Press managers, so they can think about their part, and whether they want journal staff to see this or not. And only then to make it visible.

What I want to avoid is journal staff seeing the setting, and it's also the first time they've seen a setting that is not active.  Particularly as by making this an 'a11y' group setting, it appears by default at the start of the all settings list.

Longer term, yes, I do think they should be visible but inactive and signposted as 'needs X setting on press'.  But I don't want to introduce both inactive settings and accessibility page at the same time.

StephDriver commented on src/utils/install/journal_defaults.json:
```diff
@@ -5596,5 +5596,24 @@
+        },
+        "editable_by": [
+            "journal-manager"
+        ]
```
I looked at doing this - but the file is not consistent as is - the setting attributes aren't always in the same order  - so making it consistent feels like a stand alone task.

joemull commented:
BTW, apologies this sat for so long on my plate! I did not realize it was waiting on me.

StephDriver commented:
Notes:
- removed depends on press setting
- removed migration for that setting

Next:
- replace with other logic to create same effect for a11y info only.
- rebase (to get latest clean footer)
